src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AwaitExpressionSpiller.cs

```diff
@@ -487,21 +492,33 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
             }
 
             var result = ArrayBuilder<BoundExpression>.GetInstance();
```

Comment:
>GetInstance [](start = 55, length = 11)

Can we guess the required size?

Reply:
`newList.Length` would be a fair estimate. 

---
In reply to: [162473675](https://github.com/dotnet/roslyn/pull/24273#discussion_r162473675) [](ancestors = 162473675)